### PR TITLE
ENDOC-455 7PNG + Page Update for Content Types & Content Templates

### PR DESCRIPTION
**Pull request by Lyd1aCla1r3**

### CODE REVIEW

File: vuepress/docs/next/tutorials/compose/content-types-tutorial.md [hunk 1]
```diff
@@ -1,101 +1,70 @@
 # Content Types
```
Comment (nshaw) [inline vuepress/docs/next/tutorials/compose/content-types-tutorial.md:2]: Might want to add the sidebarDepth: 2 to the header to make the page a bit more usable.

**Review by nshaw — COMMENTED** [review]
Will there be a second tutorial for the content templates or do the templates not need any changes?

File: vuepress/docs/next/tutorials/compose/content-types-tutorial.md [hunk 2]
```diff
@@ -1,101 +1,70 @@
...
+A Content Type represents or models a Content. Alternatively, a Content is an instance or specialization of a Content Type. 
 
-## Create a Content Type
-
-Creating Contents Types is fairly simple. The Content Types are managed
-through the Web CMS, its user interface is made available in the App
-Builder application.
-
--   Go to `Content → Types`
+Content Types are characterized by [Attribute Types](./content-attributes.md). Each Attribute Type consists of one or more properties, known as Attributes. An Attribute is defined by its Attribute Type.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/compose/content-types-tutorial.md:5]: Each Attribute Type consists of one or more properties, known as Attributes. An Attribute is defined by its Attribute Type. >>these sentences are redundant?

File: vuepress/docs/next/tutorials/compose/content-types-tutorial.md [hunk 3]
```diff
@@ -1,101 +1,70 @@
...
+**2. Click the `Add` button in the upper right corner.**
 
-![image](./img/ContentTypes3.png)
+![addContentType](./img/content_types1.png)
 
--   Select the **Save** button, then you’ll see
+**3. Configure the Content Type.**
 
-    -   **Content Type Information**:
+When adding a Content Type it is mandatory to enter `Code` and `Name` values consistent with the following parameter definitions:
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/compose/content-types-tutorial.md:19]: can end with "the following:"

File: vuepress/docs/next/tutorials/compose/content-types-tutorial.md [hunk 4]
```diff
@@ -1,101 +1,70 @@
...
 
-        -   **Code**. The value is pre-filled.
+- `Code`: A unique identifier of the Content Type that must be 3 uppercase letters (e.g. CNN).
 
-        -   **Name**. The name is pre-filled.
+- `Name`: A string value (50 characters max) consisting of uppercase and lowercase letters, numbers and/or special characters (e.g. Conference\_News).
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/compose/content-types-tutorial.md:23]: put '(50 characters max)' with the example, less disruption

File: vuepress/docs/next/tutorials/compose/content-types-tutorial.md [hunk 5]
```diff
@@ -1,101 +1,70 @@
...
+Field selections in the Metadata and Attributes sections are not mandatory, but to specify a meaningful Content Type you must choose the appropriate `Type` from the Attributes drop-down (e.g. 'Text'). The available Attribute Types define Attributes to characterize the Content.
 
-        -   **Page for on-the-fly publishing**
+![configureContentType](./img/content_types2.png)
 
-        -   **Default content template**
+**4. Click the `Add` button.**
 
-        -   **Default content template for lists**
+This initializes the configuration of the selected Attribute Type. Each Attribute Type requires its own configuration.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/compose/content-types-tutorial.md:31]: wondering if 'initializes' is the correct word here? reading this, just seemed like these steps just sets up the defining attribute types that define content whereas initializing involves something more complex.

File: vuepress/docs/next/tutorials/compose/content-types-tutorial.md [hunk 6]
```diff
@@ -1,101 +1,70 @@
...
-        -   **Default content template for lists**
+This initializes the configuration of the selected Attribute Type. Each Attribute Type requires its own configuration. 
 
-*Default content template* and *Default content template for lists* are used
-respectively to show the detailed view of the Content and its
-representation as an element in a list.
+### Attribute Configuration
 
--   **Attributes**
+Adding an Attribute Type will load the "Add attribute" page, where `Type` is pre-filled with your selection. Follow the steps below to configure the Attribute.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/compose/content-types-tutorial.md:35]: should "Adding an Attribute Type" be backticked? 
That sentence is a bit awkward>> "where type is prefilled with your selection"

File: vuepress/docs/next/tutorials/compose/content-types-tutorial.md [hunk 7]
```diff
@@ -1,101 +1,70 @@
...
-The list contains all the available attributes types that define which
-properties/attributes will characterize the Content.
+**1. Define the characteristics of the Attribute:**
 
-![image](./img/ContentTypes4.png)
+![configureAttributeType](./img/content_types3.png) 
 
-The Add button will start the configuration of the single attribute
-type. Every attribute type has its own configuration.
+- `Code`: It is mandatory to enter a unique name for the Attribute key (both Attributes of a 'Composite' Attribute Type may share a `Code` name). This field supports a string value (10 characters max) of uppercase and lowercase letters, numbers and/or special characters (e.g. title).
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/compose/content-types-tutorial.md:41]: too many parentheses in this section. It makes it hard to read. "both attributes of a composite..." can be a regular sentence. 

"10 characters max" can be added to the example.

File: vuepress/docs/next/tutorials/compose/content-types-tutorial.md [hunk 8]
```diff
@@ -1,101 +1,70 @@
...
 
-For example, to configure the **Text** Attribute Type, choose Text from
-the Types drop-down menu and then press the Add button.
+- `Name`: Enter a description of the Attribute. This field supports a string value (50 characters max) of uppercase and lowercase letters, numbers and/or special characters (e.g. Title).
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/compose/content-types-tutorial.md:43]: same, join parenthesis. I think it is also easier to pick out the information when it is at the end, so if anyone is scanning for the max allowed, it's easier to find at the end.

File: vuepress/docs/next/tutorials/compose/content-types-tutorial.md [hunk 9]
```diff
@@ -1,101 +1,70 @@
...
 
--   **Type:** the value is pre-filled. It represents the selected
-    attribute type.
+- The remaining fields are not mandatory and may be left empty.
 
--   **Code**: You should insert the name of the attribute. (You
-    can insert max 10 characters uppercase and lowercase letters,
-    numbers, and special characters \_ ). (e.g. title)
+:::tip
+Certain Attribute Types allow you to assign the `Role` of `jacms:title - The main title of a Content`. This informs other plugins or services that the Attribute is a title, regardless of its key, or `Code`. Entering "title" for the `Code` avoids confusion when this `Role` is selected.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/compose/content-types-tutorial.md:52]: other plugins or services >> can take out other, it makes it sound as if the subject is a plugin or service.

File: vuepress/docs/next/tutorials/compose/content-types-tutorial.md [hunk 10]
```diff
@@ -1,101 +1,70 @@
...
+:::
 
--   **Name**: You should insert the description of the attribute.
-    (You can insert max 50 characters uppercase and lowercase letters,
-    including numbers and special characters). (e.g. Title).
+**2. Click the `Continue` button.**
 
--   You should declare if the attribute is **mandatory**,
-    **searchable**, and **to be used as a filter in the list.**
+This adds the configured Attribute to the Attribute Type.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/compose/content-types-tutorial.md:57]: "configured Attribute" >> new configuration or configuration?

File: vuepress/docs/next/tutorials/compose/content-types-tutorial.md [hunk 11]
```diff
@@ -1,101 +1,70 @@
...
--   The **Roles** section it is possible to assign the *jacms:title
-    Attribute Role* to this attribute; in this way other plugins or
-    services will know that within the CNG entity this attribute is a
-    title, whatever the key is; however, for sake of clarity, the Code
-    of the attribute is Title.
+![modifyContentType](./img/content_types4.png)
 
--   The remaining fields related to **Sections** and **Validation -
-    OGNL** sections are not mandatory and can be left empty.
+- (Optional) Complete the additional configuration steps required by your Attribute Type, if applicable (the 'Text' Attribute Type requires no additional configuration).
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/compose/content-types-tutorial.md:61]: (the 'Text' Attribute Type requires no additional configuration)>> if this is optional, than all other types don't require anything additional so is this redundant?

File: vuepress/docs/next/tutorials/compose/content-types-tutorial.md [hunk 12]
```diff
@@ -1,101 +1,70 @@
...
-    of the attribute is Title.
+![modifyContentType](./img/content_types4.png)
 
--   The remaining fields related to **Sections** and **Validation -
-    OGNL** sections are not mandatory and can be left empty.
+- (Optional) Complete the additional configuration steps required by your Attribute Type, if applicable (the 'Text' Attribute Type requires no additional configuration). 
 
--   Press the **Continue** button. The Text attribute type requires no
-    additional configuration step.
+- (Optional) Add other Attribute Types to your Attribute, subject to the above configuration process.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/compose/content-types-tutorial.md:63]: does this mean an attribute type can be added to the configuration of another attribute type?

File: vuepress/docs/next/tutorials/compose/content-types-tutorial.md (hunk 2 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/tutorials/compose/content-types-tutorial.md:5]: reading it carefully and thinking it through i don't think so... the original phrasing implies that the type defines the property, not the other way around, and working through the exercise that's indeed the case

File: vuepress/docs/next/tutorials/compose/content-types-tutorial.md (hunk 4 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/tutorials/compose/content-types-tutorial.md:23]: that doesn't really make sense. the two don't go together; one is a criterion and the other is an example... i'll just add a second sentence.

File: vuepress/docs/next/tutorials/compose/content-types-tutorial.md (hunk 5 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/tutorials/compose/content-types-tutorial.md:31]: hm i think you're right about implied complexity. i think "launches" is a better choice.

File: vuepress/docs/next/tutorials/compose/content-types-tutorial.md (hunk 6 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/tutorials/compose/content-types-tutorial.md:35]: no backticks, that was my phrasing, not the title or value of anything. what is awkward? rereading it i'm not catching what stands out. i'll try to rephrase.

File: vuepress/docs/next/tutorials/compose/content-types-tutorial.md (hunk 10 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/tutorials/compose/content-types-tutorial.md:57]: ?? the Attribute you just configured is the configured Attribute...

File: vuepress/docs/next/tutorials/compose/content-types-tutorial.md (hunk 11 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/tutorials/compose/content-types-tutorial.md:61]: no... i think optional may be the wrong word here. whether or not additional configuration is required depends on which Attribute Type you're working with

File: vuepress/docs/next/tutorials/compose/content-types-tutorial.md (hunk 12 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/tutorials/compose/content-types-tutorial.md:63]: mistake! should be add other Attribute Types to your Content Type.

File: vuepress/docs/next/tutorials/compose/content-types-tutorial.md (hunk 10 above)
Comment (jyunmitch) [reply vuepress/docs/next/tutorials/compose/content-types-tutorial.md:61]: there's just a lot of 'attributes' and after awhile, they lose meaning

File: vuepress/docs/next/tutorials/compose/content-types-tutorial.md [hunk 13]
```diff
@@ -58,9 +62,9 @@ This adds the configured Attribute to the Attribute Type.
 
 ![modifyContentType](./img/content_types4.png)
 
-- (Optional) Complete the additional configuration steps required by your Attribute Type, if applicable (the 'Text' Attribute Type requires no additional configuration). 
+- Complete the additional configuration steps required by your Attribute Type, if applicable (the 'Text' Attribute Type requires no additional configuration). 
 
-- (Optional) Add other Attribute Types to your Attribute, subject to the above configuration process.
+- (Optional) Add other Attribute Types to your Content Type, subject to the above configuration process.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/compose/content-types-tutorial.md:67]: subject to the above configuration process >> subject to the configuration process above

File: vuepress/docs/next/tutorials/compose/content-types-tutorial.md [hunk 14]
```diff
@@ -16,40 +20,40 @@ Content Types are managed via the Web CMS, through a user interface provided by
...
 
-Adding an Attribute Type will load the "Add attribute" page, where `Type` is pre-filled with your selection. Follow the steps below to configure the Attribute.
+The "Add attribute" page loads once an Attribute is added and allows you to configure the Attribute. The `Type` field is pre-filled with the Attribute Type you selected when adding the Content Type.
 
 **1. Define the characteristics of the Attribute:**
 
 ![configureAttributeType](./img/content_types3.png) 
 
-- `Code`: It is mandatory to enter a unique name for the Attribute key (both Attributes of a 'Composite' Attribute Type may share a `Code` name). This field supports a string value (10 characters max) of uppercase and lowercase letters, numbers and/or special characters (e.g. title).
+- `Code`: It is mandatory to enter a unique name for the Attribute key of an Attribute Type. This field supports a string value of 10 characters or less consisting of uppercase and lowercase letters, numbers and/or special characters (e.g. title).
```
[the remark this answers is not in the document]
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/tutorials/compose/content-types-tutorial.md:45]: and/or before the last option implies "one or more of". "uppercase **and/or** lowercase numbers..." seems excessive and unnecessary. by definition the example does not need to include more than one of the options listed, and while i agree with you the example should, i'm matching the original text/pics for 6.3.2 and Next.

File: vuepress/docs/next/tutorials/compose/content-types-tutorial.md [hunk 15]
```diff
@@ -1,101 +1,74 @@
...
-        -   **Default content template for lists**
+This launches the configuration of the selected Attribute Type. Each Attribute Type requires its own configuration. 
 
-*Default content template* and *Default content template for lists* are used
-respectively to show the detailed view of the Content and its
-representation as an element in a list.
+## Attribute Configuration
 
--   **Attributes**
+Clicking "Add" loads a form to configure the Attribute. The `Type` field is pre-filled with your Attribute Type selection.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/compose/content-types-tutorial.md:39]: Add >> backticks?

**Review by jyunmitch — APPROVED** [review]

**Pull request merged**